test/test_jit_fuser_te.py

```diff
@@ -1833,6 +1833,8 @@ def eager(x, y, z: int):
     'expand',
     'expm1',
     'floor',
+    'fmod',
+    'fmod.autodiffed',
```

Comment:
Had to add both variants of `fmod` registered in OpInfo. Minor suggestion related to this, is to make errors more verbose (it should mention the op name). Currently, if the op is supported but is not registered in `works_list`, following error is raised (see: https://github.com/pytorch/pytorch/blob/master/test/test_jit_fuser_te.py#L1976)

> Expected test to fail. If it now works, move op into works_list

Even though the failure will mention the test name (`fmod_autodiffed_cpu_float32` - order may change of keywords autodiffed, cpu and float32) but the op name will be `fmod.autodiffed` (here `autodiffed` was the `variant_name` we passed to the OpInfo). Giving the op name in the error will help in general:

> f"Expected test to fail. If it now works, move {get_name(op)} into works_list"

Not a big change, but just something I felt might help.

What do you think? @mruberry @kshitij12345

Reply:
cc @Chillee; we should create an attribute like "supports_nnc" on the OpInfo, like we have `supports_grad`